prboom2/src/e6y.c

```diff
@@ -756,7 +756,7 @@ int HU_DrawDemoProgress(int force)
 
   prev_len = len;
 
-  V_FillRect(0, 0, SCREENHEIGHT - 4, len - 0, 4, 4);
+  V_FillRect(0, 0, SCREENHEIGHT - 4, len - 0, 4, (dsda_Flag(dsda_arg_heretic) || dsda_Flag(dsda_arg_hexen)) ? 255 : 4);
```

Comment:
The shorthand for this is `raven ? 255 : 4`. But I would probably add a variable because I try to avoid having value branches like this all over the place.

Take a look at https://github.com/kraflab/dsda-doom/blob/master/prboom2/src/dsda/global.c#L127 and add something like `g_demo_progress_color`.